opba-db/src/test/java/de/adorsys/opba/db/CsvConvertTest.java

```diff
@@ -0,0 +1,55 @@
+package de.adorsys.opba.db;
+
+import com.google.common.io.Files;
+import com.google.common.io.Resources;
+import lombok.SneakyThrows;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
+import java.io.File;
+import java.util.List;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+
+@Disabled
+public class CsvConvertTest {
+    private static final String BANKS_SOURCE = "migration/migrations/banks.csv";
+    private static final String BANKS_ACTION_SQL = "bank_action_sql.txt";
+    private static final String BANKS_SUB_ACTION_SQL = "bank_sub_action_sql.txt";
+
+    @Test
+    @SneakyThrows
+    public void convertToDbCsv() {
+        int bankId = 1;
+        int bankActionId = 1;
```

Comment:
We should configure starting value for `bankActionId` using an environment variable

Reply:
a property was created